project/src/test/java/com/google/sps/EventSpannerTasksTest.java

```diff
@@ -126,6 +126,7 @@ public void testEventCreationDoPost() throws Exception {
     request.addParameter("time", TIME);
     request.addParameter("description", DESCRIPTION);
     request.addParameter("location", LOCATION);
+    request.addParameter("interests", "Tech");
```

Comment:
Do you need to assert something equals to "Tech" below?

Reply:
done